### PR TITLE
CB-13414: Add e2e tests for re-sizing of the datalake.

### PR DESCRIPTION
**Pull request by kkalvagadda1**
E2E test for datalake resize functionality.

### CODE REVIEW

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/sdx/SdxResizeTests.java [hunk 1]
```diff
@@ -35,34 +41,65 @@
...
+    private SdxInternalTestDto validateCrn(AtomicReference<String> originalCrn, SdxInternalTestDto dto) {
+        String newCrn = dto.getResponse().getCrn();
+        Log.log(LOGGER, format(" New crn: %s ", newCrn));
+        if (!newCrn.equals(originalCrn.get())) {
+            throw new TestFailException(" The stack CRN has changed to: " + newCrn + " instead of: " + originalCrn.get());
+        }
+        return dto;
+    }
+
+    private SdxInternalTestDto validateShape(AtomicReference<String> originalCrn, SdxInternalTestDto dto) {
```
Comment (aszegedi) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/sdx/SdxResizeTests.java:88]: Here the `expectedShape` or based on previous pattern `originalShape` should be. However I cannot see the later usage of this parameter.

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/sdx/SdxResizeTests.java [hunk 2]
```diff
@@ -35,34 +41,65 @@
...
+        SdxClusterShape newShape = dto.getResponse().getClusterShape();
+        Log.log(LOGGER, format(" New shape: %s ", newShape.name()));
+        if (!SdxClusterShape.MEDIUM_DUTY_HA.equals(newShape)) {
```
Comment (aszegedi) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/sdx/SdxResizeTests.java:91]: I think we should use the `expectedShape` parameter for the validation/comparison. If the default E2E SDX `clusterShape` or `internalClusterShape` has been changed in the [application.yml](https://github.com/hortonworks/cloudbreak/blob/master/integration-test/src/main/resources/application.yml#L75-L76) test case won't fail.

**Review by aszegedi — CHANGES_REQUESTED** [review]

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/sdx/SdxResizeTests.java (hunk 2 above)
Comment (kkalvagadda1) [reply integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/sdx/SdxResizeTests.java:91]: resize functionality by default changed the shape of datalake to medium duty.

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/sdx/SdxResizeTests.java (hunk 1 above)
Comment (kkalvagadda1) [reply integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/sdx/SdxResizeTests.java:88]: that code was stale. removed it.

**Review by aszegedi — APPROVED** [review]

**Review by biharitomi — APPROVED** [review]

**Pull request merged**